liveproxy/argparser.py

```diff
@@ -66,13 +56,13 @@ def file_output_list(value):
     '--host',
     metavar='HOST',
     type=ip_address,
-    default='127.0.0.1',
+    default='localhost',
```

Comment:
won't work like this

`liveproxy: error: argument --host: invalid ip_address value: 'localhost'`

Reply:
I never actually tried the default :thinking:
I suppose actually then there should be no validation on the `host` variable at all then, because supplying a hostname should be valid

Reply:
Would you be in favor of removing the type check here? Any value should be passed to `getaddrinfo` before `bind` assuming the http library is sane

Reply:
> Would you be in favor of removing the type check here?

sure, could be done

but let `127.0.0.1` as **default** so it don't change anything if `localhost` is not set correctly

Reply:
The logic here was that binding `::1` and `127.0.0.1` would be covered by `localhost`. I'm not sure of any other way to bind both in a single bind call. I suppose one could assume binding to `lo0` but that's equally as bad.